src/containers/GalleryView.js

```diff
@@ -16,6 +17,43 @@ const mapStateToProps = (state, { window }) => (
   }
 );
 
+/**
+ * Styles to be passed to the withStyles HOC
+ */
+const styles = theme => ({
+  currentCanvas: {
+    border: `2px solid ${theme.palette.secondary.main}`,
+    padding: '3px',
+  },
+  galleryContainer: {
+    flex: '1',
+    overflowX: 'hidden',
+    overflowY: 'scroll',
+    padding: '50px 0 50px 20px',
+  },
+  galleryViewItem: {
+    '&:focus': {
+      outline: 'none',
+    },
+    '&:hover': {
+      border: `2px solid ${theme.palette.secondary.main}`,
+      padding: '3px',
+      transform: 'scale(1.05)',
+      transition: '.1s transform ease-out',
+    },
+    boxSizing: 'border-box',
+    cursor: 'pointer',
+    display: 'inline-block',
+    height: '160px',
+    margin: '10px 5px',
+    maxWidth: '100px',
+    overflow: 'hidden',
+    padding: '5px',
```

Comment:
I'm not sure if the padding/margins here should be using theme variables or not (I'm just moving it from index.scss here).  If anybody thinks that it would make sense to use the spacing vars or anything, let me know.

Reply:
I'd certainly be in favor of using the theme spacing (unless there's a strong reason we need 1px more spacing in gallery view than everywhere else we're adding padding + margins)?